contracts/ccip/interfaces/IGovernanceCCIPRelay.sol

```diff
@@ -8,6 +8,19 @@ import {IRouterClient} from "@chainlink/contracts-ccip/src/v0.8/ccip/interfaces/
  * @dev Interface for the GovernanceCCIPRelay contract.
  */
 interface IGovernanceCCIPRelay {
+  /// @notice Emitted when a governance message is relayed.
+  /// @param target The target address of the execution.
+  /// @param payload The calldata payload executed.
+  event MessageRelayed(address indexed target, bytes payload);
+
+  /// @notice Emitted when a new destination chain is added.
+  /// @param chainSelector The CCIP chain selector of the destination chain.
+  /// @param receiver The address of the receiver contract on the destination chain.
+  event DestinationChainAdded(
+    uint64 indexed chainSelector,
+    address indexed receiver
+  );
+
   /// @notice Emitted when the destination receiver is updated.
   /// @param oldReceiver The previous destination receiver address.
   /// @param newReceiver The new destination receiver address.
```

Comment:
@voith I think that you should **also** track the `uint64 indexed chainSelector` for which the `DestinationReceiverUpdated` has been triggered for

Reply:
done